### ISSUE TITLE
Use planets for all server iconography

### ISSUE THREAD

**Issue by nclm** [mastodon/joinmastodon#290]
All the nice elephant illustrations use planets to represent servers, as does the introduction animated video. I think it’s a great metaphor :)

Then there are these two icons: 
https://joinmastodon.org/_next/static/media/servers.859ae0fe.svg
https://joinmastodon.org/_next/static/media/move-servers.c39ef9cb.svg

Suddenly, servers are shown as hardware. Those are very technical, more like what you will see from a hosting company. Not particularly clear to people unfamiliar with data centres, and not very welcoming. They are also inconsistent with how servers are shown in other places. I believe those should depict planets instead?

Additionally, the decentralised icon could also show different planets connecting to each other to keep the concept clear all the way:
https://joinmastodon.org/_next/static/media/decentralized.9e11e489.svg

Basically this is about sticking to the visual metaphor in all places :)
